README.md

```diff
@@ -1,26 +1,26 @@
 Baraja Doctrine database
 ========================
 
-Simple easy to use, maximal performance database layer with connection to Doctrine and support for Nette 3.0.
+A simple and easy to use, maximum performance database layer with connection to Doctrine, which allows you to use all the advantages of OOP and also has support for Nette 3.0.
 
-This package install Doctrine automatically to your project with stable run.
+This package automatically installs Doctrine (also setting everything up in the configuration) to your project and runs stably.
```

Comment:
Please add something like: "If you have PackageRegistrator" (automatic installation process).